nodejs/src/api_object.js

```diff
@@ -1,19 +1,28 @@
 import got from 'got'
-import {ApiCommon} from '../api_common.js'
-import {Input} from '../utils.js'
+import {ApiCommon} from './api_common.js'
+import {Input} from './utils.js'
 
 /**
  * The ApiObject uses the got library for REST transations:
  *   https://www.npmjs.com/package/got
  */
 export class ApiObject extends ApiCommon {
   constructor(api_config, access_token) {
-    super();
+    super(api_config);
     this.api_uri = api_config.api_uri;
-    this.api_config = api_config;
     this.access_token = access_token;
   }
 
+  // This method extracts the data container from the v3 response body
+  // and returns the v5 response body without modification.
+  extract(body) {
+    if (this.api_config.version == 'v3') {
```

Comment:
`return (this.api_config.version === 'v3') ? body.data : data;`

Reply:
done.